app/Http/Controllers/AuthPatient/RegisterController.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace App\Http\Controllers\AuthPatient;
```

Comment:
Ne legyen inkább külön Patient vagy külön Auth namespace?

Reply:
jobb lenne tényleg,  nem értem amúgy a Laravel miért nem modulárisan strukturálja mappáit, (bár van erre package)
Csinálhatjuk úgy következetesen, az egyes  egységek (controller, Services atv) alá külön namespacekben legyenen az eltérő Model-t érintő classok
endobabyben is így csinálom kb:
https://github.com/delocalzrt/endobaby-backend/tree/develop/app/Services